### DISCUSSION TITLE
What causes "crashing" attention?

### DISCUSSION THREAD

**Discussion by rezareza007** [coqui-ai/TTS#1944] [General Q&A]
I've trained Tacotron2 on a custom dataset and for some sentences, the generated audio has a long-high pitch sound at the end. The attention for such files "crashes" on the horizontal axis, long before reaching the right bottom corner and the horizontal axis is much longer than the vertical one. What causes this?

**Comment by rezareza007** [comment]
Hello!

**Reply by p0p4k** [reply]
Could be that the model is still learning. How many steps into the training are you at?

**Reply by rezareza007** [reply]
More than 50000. Letting training to continue causes overfitting.
